slick/src/main/scala/slick/jdbc/GetResult.fm

```diff
@@ -3,8 +3,10 @@ package slick.jdbc
 import java.sql.{Date, Time, Timestamp}
 import slick.SlickException
 import slick.util.TupleSupport
+import scala.annotation.implicitNotFound
 
 /** Basic conversions for extracting values from PositionedResults. */
+@implicitNotFound("No GetResult found for type ${r"${T}"}. Try to implement an implicit GetResult for this type.")
```

Comment:
I think this is a little more helpful,and in the code base we have made use of `implicitNotFound` for other part,so this could be consistence too.